### PR TITLE
#12102 chat.py example changed to byte concat instead of decode/encode

### LINKED ISSUE #12102: Avoid encode/decode in chat.py for better readablity
As discussed in [this comment](https://github.com/twisted/twisted/pull/12070#discussion_r1442784443), it's better to use byte concat as mentioned in order of better readability in [docs/core/howto/listings/servers/chat.py:35](https://github.com/twisted/twisted/pull/12070/files/c59c93ec644a17e0f3a1752ca9ceca31a27a9f5e#diff-0923ff3db530a2e5d28ea8cc2b3a8f91f399792786772c541bf9edf7a0c50126)
```python
message = b'<' + self.name + b'> ' + message
```

### PR DESCRIPTION
**Pull request by snosratiershad**
## Scope and purpose

Fixes #12102

### CODE REVIEW

**Comment by snosratiershad** [comment]
please review

File: src/twisted/newsfragments/12097.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+The documentation code for chat example changed for readability
```
Comment (adiroiban) [inline src/twisted/newsfragments/12097.doc]: The IRC server example found in the documentation was updated for readability.

**Review by adiroiban — APPROVED** [review]
Looks good. Thanks.

Only a minor suggestion related to the release notes.

**Comment by snosratiershad** [comment]
@adiroiban, Done.

**Comment by adiroiban** [comment]
Thanks for the update. I have enabled auto-merge. I hope it will land soon. Thanks again!

**Pull request merged**